### PR TITLE
Update and finalise user guide and README

### PR DESCRIPTION
**Pull request by SukiTsang**

### CODE REVIEW

**Comment by advaypal** [comment]
Travis is failing

**Comment by SukiTsang** [comment]
Need to re-run the travis

File: docs/UserGuide.md [hunk 1]
```diff
@@ -150,8 +205,8 @@ Format: `find KEYWORD` or `find t/TAG`
 Examples:
 * `find meeting`<br>
   Returns tasks having name `meeting` 
-* `find tag cs2103`<br>
-  Returns tasks having tag `cs2103`
+* `find cs2103`<br>
+  Returns tasks having name `cs2103`
```
Comment (Skaty) [inline docs/UserGuide.md]: Name or substring.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -133,13 +187,14 @@ Example:
...
 Shows a list of tasks and events in the todo list.<br>
 Format: `list`
 
 #### Finding all tasks and events containing keyword in their name: `find`
 Finds tasks which have names containing any of the given keywords.<br>
-Format: `find KEYWORD` or `find t/TAG`
+Format: `find KEYWORD`
 
 > * The search is case insensitive. e.g `meeting` will match `Meeting`
 > * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
```
Comment (Skaty) [inline docs/UserGuide.md:201]: Add the fact that you can search substrings e.g. `mee` will match `meeting`.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -2,9 +2,31 @@
...
+  * [Help](#viewing-help--help)
+  * [Change-to](#change-storage-location--change-to)
+  * [Exit](#exiting-the-program--exit)
+* [FAQ](#faq)
 * [Command Summary](#command-summary)
 
+<!-- @@author A0153467Y -->
+## Introduction
+
+Nowadays, everyone got so many tasks to do. Have you ever miss your deadlines or forget your schedule? MESS are here to help you! MESS is a to-do list application which reminds you of your tasks or events by showing your list of tasks on the application. No need to afraid about forgetting your tasks anymore! You can easily use MESS by typing in one line of command. MESS can be used offline and even in your office computer. It helps you manage and organise your tasks and so as your time.
```
Comment (Skaty) [inline docs/UserGuide.md]: Need to fix the grammar a little.

**Review by Skaty — CHANGES_REQUESTED** [review]
Please see comments and reply/make changes.

File: docs/UserGuide.md (hunk 3 above)
Comment (SukiTsang) [reply docs/UserGuide.md]: Please check again, thanks. My grammar is not so good.

**Review by Skaty — APPROVED** [review]
:+1:, needs final fixing later on.

**Pull request merged**